spec/web/controllers/auth_controller_spec.exs

```diff
@@ -37,10 +37,10 @@ defmodule Squadster.Web.AuthControllerSpec do
 
             it "redirects to front-end with user info" do
               redirect_url = success_conn() |> AuthController.callback(%{}) |> redirected_to(302)
-              expect(redirect_url =~ "last_name")
-              expect(redirect_url =~ "uid")
-              expect(redirect_url =~ "auth_token")
-              expect(redirect_url =~ "image_url")
+              expect(redirect_url =~ "last_name") |> to(be true)
+              expect(redirect_url =~ "uid") |> to(be true)
+              expect(redirect_url =~ "auth_token") |> to(be true)
+              expect(redirect_url =~ "image_url") |> to(be true)
```

Comment:
How did it work?? 0_o

Reply:
This test always passed because had no expectations :))))))))))))))))